e2e/baseFixtures.js

```diff
@@ -170,5 +170,20 @@ exports.test = base.test.extend({
         }
     }
 });
+
+/**
+ * Takes a readable stream and returns a string.
+ * @param {ReadableStream} readable - the readable stream
+ * @return {Promise<String>} the stringified stream
+ */
+exports.streamToString = async function (readable) {
```

Comment:
Let's move this to `pluginFixtures` instead. Each function in `baseFixtures` is tied to an existing RFE on GitHub, so the idea is that it will eventually go away once it's all implemented into playwright